README.md

```diff
@@ -32,11 +51,11 @@ otherLogger.info('Hi, %o', {world: 'Earth', age: 9999999999})
 // this methods are applied to both logger and otherLogger
```

Comment:
Logger is `undefined` in example
https://github.com/ilya-babanov/lggr/pull/3/files#diff-04c6e90faac2675aa89e2176d2eec7d8R41